src/extension/ogcapi/web-ogcapi/src/test/java/org/geoserver/web/ogcapi/OgcApiServiceDescriptionProviderTest.java

```diff
@@ -28,6 +29,10 @@ public void testBasic() {
         assertEquals("TestCaseServiceType", provider.getServiceType());
         assertEquals("OGCAPI-TestCase", provider.getServiceName());
         assertEquals("TestCase", provider.getSpecificServiceType());
+        assertTrue(
```

Comment:
I'm confused as to what this test is checking. What's the point of having a list with the same value twice?
And how does it relate to the issue reported?
A test for the Pr should replicate the failure in the home page and check it's gone after the fix...

Reply:
Also, shouldn't the test check for serviceType and specificServiceType?
It seems to be passing by accident.

Reply:
Yes test was not correct, added an explicit "FakeServiceDescriptorProvider" to use with CapabilitiesHomePagePanelTest to ensure the skipped service capabilities are not displayed.